tests/verifyDashboards_test.js

```diff
@@ -0,0 +1,26 @@
+Feature('PT Summary');
+const { dashboardPage } = inject();
```

Comment:
verifyDashboards_test, the name of the tests is very confusing, rather make it verifyPTSummaryPanels_test.js because that's exactly what the feature is about.